kubernetes/charts/weblogic-operator/values.yaml

```diff
@@ -191,3 +191,9 @@ externalServiceNameSuffix: "-ext"
 # cluster size.
 # The default value is true.
 clusterSizePaddingValidationEnabled: true
+
+# tokenReviewAuthentication specifies whether the the operator's REST API should use
+#   1. Kubernetes token review API for authenticating users, and
+#   2. Kubernetes subject access review API for authorizing a user's operation (get, list,
+#      patch, etc) on a resource.
+#tokenReviewAuthentication: false
```

Comment:
"The default value is false." would be a good thing to note.

Reply:
This leaves ambiguous the meaning of "false." Would you please update this to describe that when the value is false that the caller's bearer token will be delegated so that the update to the Domain resource is done using the caller's privileges?

Reply:
updated with description.